packages/block-editor/src/components/image-size-control/test/index.js

```diff
@@ -272,15 +233,12 @@ describe( 'ImageSizeControl', () => {
 				/>
 			);
 
-			const button = screen.getByRole( 'button', {
+			const button = screen.getByRole( 'radio', {
 				name: '50%',
-				pressed: false,
 			} );
 
 			await user.click( button );
 
-			expect( button ).toHaveClass( 'is-pressed' );
```

Comment:
Should we check for `expect( button ).toBeChecked()` here instead of removing the previous check?

Reply:
I agree. Furthermore, although this is not required, I think it might be a good idea to combine the following two tests:

- updates height and width attributes on selection
- updates height and width inputs on selection

<details><summary>Details</summary>

```javascript
	describe( 'image size percentage presets', () => {
		it( 'updates height and width on selection', async () => {
			const user = userEvent.setup();

			render(
				<ImageSizeControl
					imageHeight="100"
					imageWidth="201"
					onChange={ mockOnChange }
				/>
			);

			const button = screen.getByRole( 'radio', {
				name: '50%',
			} );

			await user.click( button );

			// Both attributes are set to the rounded scaled value.
			expect( mockOnChange ).toHaveBeenLastCalledWith( {
				height: 50,
				width: 101,
			} );

			expect(
				screen.getByRole( 'spinbutton', { name: 'Height' } )
			).toHaveValue( 50 );
			expect(
				screen.getByRole( 'spinbutton', { name: 'Width' } )
			).toHaveValue( 101 );
		} );
	} );
```

</details>